src/ResourceManagement/NotificationHubs/Microsoft.Azure.Management.NotificationHubs/generated.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=0.14.0-Nightly20160125
```

Comment:
father change of the version delete your generated folder and re-run this script.